amanuensis/resources/fence/__init__.py

```diff
@@ -0,0 +1,53 @@
+
+import json
+from inspect import signature
+
+import requests
+from cdislogging import get_logger
+
+from amanuensis.auth.auth import get_jwt_from_header
+from pcdcutils.signature import SignatureManager
+from pcdcutils.errors import NoKeyError
+from pcdcutils.helpers import encode_str
+
+logger = get_logger(__name__)
+
+
+def fence_get_users(usernames, config):
+    '''
+    amanuensis sends a request to fence for a list of user ids 
+    matching the supplied list of user email addresses
+    '''
+    queryBody = {
+        'usernames': usernames
+    }
+
+    try:
+        # sending request to Fence
+        url = config['FENCE'] + "/admin/users/selected"  
```

Comment:
https://github.com/chicagopcdc/amanuensis/blob/PEDS-436_send_emails_when_message_sent/amanuensis/config-default.yaml

Can you add "FENCE" to the config-default file. It can be empty, but just to have the key there